Engine/SerializerPlugins/ObjectSerializer.cs

```diff
@@ -592,7 +592,14 @@ public override bool Deserialize(XElement element, ITypeData t, Action<object> s
             }
             catch(Exception ex)
             {
-                Serializer.PushError(element, $"Object value was not read correctly.", ex);
+                if (ex is TargetInvocationException tarEx)
+                {
+                    Serializer.PushError(element, tarEx.InnerException.Message, tarEx);
```

Comment:
Maybe this would be better?

```suggestion
                    Serializer.PushError(element, tarEx.InnerException.Message, tarEx.InnerException);
```